src/frontend/AppContainerWrapper.tsx

```diff
@@ -142,7 +178,7 @@ const AppContainerWrapper = () => {
                 experiments.onboarding
               );
 
-              const loadedRouteName = getRouteName(loadedNavState);
+              const loadedRouteName = getRouteName(loadedNavState!);
```

Comment:
generally not a fan of this operator. I would either update `loadSavedNavState` so that it definitively returns `NavigationState | null` (right now it returns `NavigationState | null | void`